### PR TITLE
post: ng 4.4

### PR DESCRIPTION
**Pull request by cexbrayat**
Angular 4.4 dropped yesterday. Not much to say about it, but I wrote about it anyway.
Release target is Monday morning if you have time to review it, but we can publish later if you can't.

### CODE REVIEW

**Comment by cexbrayat** [comment]
I'm currently working on the Pro Pack to have an up-to-date version

File: _posts/2017-09-18-what-is-new-angular-4.4.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      templateUrl: './home.component.html',
+      preserveWhitespaces: false
+    })
+    export class HomeComponent implements OnInit, OnDestroy {
+
+If you really want a whitespace to be kept,
+you can use a special entity called `&ngsp;`.
+It looks like `&nbsp;` with a typo, but it is not:
+it is a special character that the Angular compiler will transform in a whitespace.
+Note that it will only keep one whitespace, even if add several ones, like `&ngsp;&ngsp;`.
```
Comment (jnizet) [inline _posts/2017-09-18-what-is-new-angular-4.4.md]: even if you add several consecutive ones, like...

File: _posts/2017-09-18-what-is-new-angular-4.4.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    <div ns-ninja #foo="ninja">
+    <!-- or -->
+    <div ns-ninja #foo="superNinja">
+
+# Summary
+
+That's all for this release: the next important one will be 5.0!
+
+If you missed, there have been more movement on Angular CLI lately.
```
Comment (jnizet) [inline _posts/2017-09-18-what-is-new-angular-4.4.md]: The Angular CLI also had interesting changes lately. Check out out blog posts if you missed them:

File: _posts/2017-09-18-what-is-new-angular-4.4.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+Check out our blog posts if you missed them:
+
+- [Angular CLI 1.3](/2017/08/10/angular-cli-1.3/)
+- [Angular CLI 1.4](/2017/09/14/angular-cli-1.4/)
+
+In the meantime, all our materials ([ebook](https://books.ninja-squad.com/angular), [online training (Pro Pack)](https://angular-exercises.ninja-squad.com/) and [training](http://ninja-squad.com/training/angular)) are up-to-date with these changes if you want to learn more!
```
Comment (jnizet) [inline _posts/2017-09-18-what-is-new-angular-4.4.md]: I'd remove "In the meantime".

**Pull request merged**